submitted_models/ctu_cras_norlab_absolem_sensor_config_1/CMakeLists.txt

```diff
@@ -3,18 +3,18 @@ project(ctu_cras_norlab_absolem_sensor_config_1)
 
 set(CMAKE_CXX_STANDARD 17)
 
-find_package(ignition-gazebo2 REQUIRED)
+find_package(ignition-gazebo3 REQUIRED)
```

Comment:
I don't know if you actually intended to modify this submitted model here.

Reply:
Yes, it's needed in order to compile against ignition-citadel.